### DISCUSSION TITLE
Export schema from directus cloud

### DISCUSSION THREAD

**Discussion by josias-r** [directus/directus#13733] [Archive]
Is there any way to export the schema from a directus cloud project?

It's currently not possible through the UI, and without CLI access it therefore seems impossible overall.

I did find a schema migrate support request, but that one is only TO a cloud instance. Apparently I cannot export the schema to use it with a self hosted instance

**Comment by benhaynes** [comment]
We are working on direct database access, as well as enabling schema migrations via the API (not just the CLI). This work should be released soon!

**Reply by josias-r** [reply]
Awesome, then I will look forward to that release 🥳
